### ISSUE TITLE
[Crash]: 

### ISSUE THREAD

**Issue by akgoakgo** [OpenTTD/OpenTTD#11298]
### Version of OpenTTD

13.3 windows 11 home

### Steps to reproduce

Made changes to the terrain next to a station and a iron ore mine.

### Upload crash files

[crash20230914202504.zip](https://github.com/OpenTTD/OpenTTD/files/12613107/crash20230914202504.zip)

**Comment by James103** [comment]
From the crash screenshot, this is likely a duplicate of https://github.com/OpenTTD/OpenTTD/issues/11043. Will you please update to 13.4 or later?

**Comment by glx22** [comment]
```
>	[Cadre en ligne] openttd.exe!HasBit(const RailTypes) Ligne 105	C++
 	openttd.exe!HasRailtypeAvail(const Owner company, const RailType railtype) Ligne 188	C++
 	openttd.exe!ValParamRailtype(const RailType rail) Ligne 208	C++
 	openttd.exe!ShowBuildRailToolbar(RailType) Ligne 860	C++
 	openttd.exe!MenuClickBuildRail(int index) Ligne 924	C++
 	openttd.exe!MainToolbarWindow::OnDropdownSelect(int widget, int index) Ligne 2040	C++
 	openttd.exe!Window::OnDropdownClose(Point pt, int widget, int index, bool instant_close) Ligne 305	C++
 	openttd.exe!DropdownWindow::Close() Ligne 200	C++
 	openttd.exe!SetFocusedWindow(Window * w) Ligne 472	C++
 	openttd.exe!Window::InitializeData(int window_number) Ligne 1444	C++
 	openttd.exe!Window::FinishInitNested(int window_number) Ligne 1794	C++
 	openttd.exe!NewsWindow::NewsWindow(WindowDesc * desc, const NewsItem * ni) Ligne 311	C++
 	openttd.exe!ShowNewspaper(const NewsItem * ni) Ligne 621	C++
 	openttd.exe!MoveToNextNewsItem() Ligne 741	C++
 	openttd.exe!StateGameLoop() Ligne 1429	C++
 	openttd.exe!GameLoop() Ligne 1513	C++
 	openttd.exe!VideoDriver::GameLoop() Ligne 38	C++
 	openttd.exe!VideoDriver::GameThread() Ligne 47	C++
 	openttd.exe!StartNewThread::__l3::<lambda>(const char * name, void(*)(VideoDriver *) && F, VideoDriver * && <A_0>) Ligne 65	C++
 	[Code externe]	
 	openttd.exe!thread_start<unsigned int (__cdecl*)(void *),1>(void * const parameter) Ligne 97	C++
 	[Code externe]	
```
Indeed duplicate of #11043, fixed in version 13.4.

**Comment by akgoakgo** [comment]
Thank you!
